python/orca/src/bigdl/orca/data/shard.py

```diff
@@ -175,28 +174,42 @@ def __init__(self,
             self.rdd.cache()
         if self.eager:
             self.compute()
+            self.is_lazy = False
+        else:
+            self.is_lazy = True
         self.type = {}
         if class_name:
             self.type['class_name'] = class_name
 
     @classmethod
-    def _create(cls,
+    def lazy(cls,
```

Comment:
@jason-dai Using LazySparkXShards would unexpectedly caused uncache for the original xshards, reason not found. Thus removed this subclass and achieve the lazy functionality directly in SparkXShards.
Take a look. See more details here: https://github.com/analytics-zoo/orca/issues/136#issuecomment-1337037891